ldhooks/hooks.go

```diff
@@ -0,0 +1,93 @@
+package ldhooks
+
+import (
+	"context"
+
+	"github.com/launchdarkly/go-sdk-common/v3/ldreason"
+)
+
+// Implementation Note: The Unimplemented struct is provided to simplify hook implementation. It should always
+// contain an implementation of all series and handler interfaces. It should not contain the Hook interface directly
+// because the implementer should be required to implement Metadata.
+
+// A Hook is used to extend the functionality of the SDK.
+//
+// In order to avoid implementing unused methods, as well as easing maintenance of compatibility, implementors should
+// compose the `Unimplemented`.
+//
+//	type MyHook struct {
+//	  Unimplemented
```

Comment:
```suggestion
//	  ldhooks.Unimplemented
```
Mainly because that is exactly what they will need to type so they can copy/paste